requirements_dev.txt

```diff
@@ -4,7 +4,7 @@ bump2version
 coverage
 flake8
 flake8-rst-docstrings
-ipython<8.0  # this needed when testing see #1005
+ipython
```

Comment:
Should we be pinning this above 8.2?

Reply:
I believe the faulty IPython has been removed from this universe! I can't see it neither in conda nor in the github's tags. (8.0)